app/models/assignment.rb

```diff
@@ -661,7 +661,7 @@ def get_detailed_csv_report
     students = Student.all
     # determine whether to use flexible criterion or rubric
     is_flexible = self.marking_scheme_type == MARKING_SCHEME_TYPE[:flexible]
```

Comment:
@david-yz-liu I will try to delete it in my second pull request. The reason why I do not delete it here is that 'is_flexible' is used in other places in this method, and deleting it will need some method refactoring instead of using get_criteria.